web/src/components/FilingsCalendar.test.tsx

```diff
@@ -396,42 +396,6 @@ describe("<FilingsCalendar />", () => {
       expect(screen.queryByTestId("get-tax-access")).not.toBeInTheDocument();
     });
 
-    it("hides button on filings calendar when feature flag is disabled", () => {
```

Comment:
i agree this should be removed but in the future i think we should avoid putting disparate things in one refactor commit